Habitica/src/main/java/com/habitrpg/android/habitica/models/inventory/Animal.java

```diff
@@ -15,6 +15,8 @@
 
     void setType(String type);
 
+    String getTranslatedType(Context c);

```

Comment:
I forgot that Animal is an interface. Can you try moving the `getTranslatedType` method to a [kotlin extension](https://kotlinlang.org/docs/reference/extensions.html) for the Animal interface? That way, the method would only have to be implemented once for both the pets and mounts classes.